drv/lpc55-swd/src/main.rs

```diff
@@ -74,6 +74,7 @@ enum Trace {
     WriteCmd,
     None,
     AckErr(Ack),
+    ParityFail(u32, u16),
```

Comment:
Speaking as a person who may want to debug this later, could you name these fields? From the code I _think_ that these would be reasonable names:
```rust
ParityFail {
    data: u32,
    received_parity: bool,
}
```
(also does `parity` need to be a u16?)